packages/checkbox/stories/checkbox.stories.ts

```diff
@@ -30,6 +30,12 @@ export const readonly = (): TemplateResult => {
     `;
 };
 
+export const WithLabel = (): TemplateResult => {
+    return html`
+        <sp-checkbox label="Checkbox Label">Checkbox</sp-checkbox>
+    `;
+};
+
 export const Indeterminate = (): TemplateResult => {
```

Comment:
Without an explicit label, the Checkbox will be accessible and uses the text between the tags. With the label you can explicitly set the `aria-label` of the underlying `input` tag to support checkboxes without a provided label text between tags. For the table use case this is important, since there we only have empty sp-checkbox tag